packages/lesswrong/components/posts/PostsPage/AudioToggle.tsx

```diff
@@ -0,0 +1,81 @@
+// TODO: Import component in components.ts
+import React, { useEffect, useMemo } from 'react';
+import { registerComponent, Components } from '../../../lib/vulcan-lib';
+import { useCookiesWithConsent } from '@/components/hooks/useCookiesWithConsent';
+import moment from 'moment';
+import { PODCAST_TOOLTIP_SEEN_COOKIE } from '@/lib/cookies/cookies';
+import classNames from 'classnames';
+import { isLWorAF } from '@/lib/instanceSettings';
+import { isFriendlyUI } from '@/themes/forumTheme';
+
+const PODCAST_ICON_SIZE = isFriendlyUI ? 22 : 24;
+// some padding around the icon to make it look like a stateful toggle button
+const PODCAST_ICON_PADDING = isFriendlyUI ? 4 : 2
+
+const styles = (theme: ThemeType) => ({
+  togglePodcastContainer: {
+    alignSelf: 'center',
+    color: theme.palette.text.dim3,
+    height: PODCAST_ICON_SIZE,
+  },
+  audioIcon: {
+    width: PODCAST_ICON_SIZE + (PODCAST_ICON_PADDING * 2),
+    height: PODCAST_ICON_SIZE + (PODCAST_ICON_PADDING * 2),
+    transform: `translateY(-${PODCAST_ICON_PADDING}px)`,
+    padding: PODCAST_ICON_PADDING
+  },
+  audioNewFeaturePulse: {
+    top: PODCAST_ICON_PADDING * 1.5
+  },
+  audioIconOn: {
+    background: theme.palette.icon.dim05,
+    borderRadius: theme.borderRadius.small
+  }
+});
+
+const AudioToggle = ({classes, post, toggleEmbeddedPlayer, showEmbeddedPlayer}: {
```

Comment:
Are there any changes here that aren't just factoring this out of `PostsPagePostHeader`?

Reply:
Not that I know of.